telegram/ext/jobqueue.py

```diff
@@ -379,7 +376,7 @@ def __init__(self,
         self.context = context
         self.name = name or callback.__name__
 
-        self._repeat = repeat
+        self._repeat = None
```

Comment:
What prompted this change?

Reply:
To be honest it was only for aesthetic reasons. Since all the other properties were also being set this way—i.e. initialising the internal attribute to `None` and then triggering the `<property>.setter` method by assigning to the "public" attribute (e.g. `self._interval = None; self.interval = interval`)—, I changed it to make it consistent. But now that I think of it, it's more than just aesthetic, because this way we ensure that any checks done by the `<property>.setter` method get executed at initialisation too.

Reply:
oh yeah i see it now, thanks 👍

Reply:
Actually, I don't understand where it is being set now. Or something is wrong with my diff?

Reply:
oic. yes, it's ok.